app/controllers/task_controller.rb

```diff
@@ -1,33 +1,77 @@
 class TaskController < ApplicationController
   def index
     @tasks = Task.all
-  end
 
-  def show
-    @id = params[:id]
-    @task = Task.find(@id)
+    @people = Person.all
+
+    render :index
   end
 
+
   def new
-    @new_task = Task.new
+    @task = Task.new
+
+    @people = Person.all
   end
 
   def create
-    @new_task = Task.new(create_params[:task])
-    @new_task.save
+    @task = Task.new(create_params[:task])
+    @task.save
 
     redirect_to '/'
   end
 
+  def show
+    @task = Task.find(params[:id])
+
+    @person_id = @task.person_id
```

Comment:
Here is another instance where you could use a local variable rather than an instance variable since this is not used in the view.